src/platform/i386/boot_comp.c

```diff
@@ -112,7 +115,7 @@ kern_boot_comp(void)
         struct captbl *ct;
         unsigned int i;
 	u8_t *boot_comp_captbl;
-	pgtbl_t pgtbl = (pgtbl_t)chal_va2pa(&boot_comp_pgd);
+	pgtbl_t pgtbl = (pgtbl_t)chal_va2pa(&boot_comp_pgd), local_pgd;
```

Comment:
I don't think this variable is being used at all.  Please always do a `git diff` before committing.  Clean up things like this.